client/my-sites/post-type-list/post-actions-ellipsis-menu/qrcode.jsx

```diff
@@ -0,0 +1,61 @@
+			fill-color="#000000"
+		/>
+	</svg>
+);
+
+function PostActionsEllipsisMenuQRCode( { translate, status, url } ) {
+	const [ showQRCode, setShowQRCode ] = useState( false );
+
+	const generateQRCode = () => {
+		setShowQRCode( true );
+	};
+
+	if ( ! url || ! status ) {
+		return null;
+	}
+
+	// The QR option will be available only for `publish` status
+	if ( status !== 'publish' ) {
+		return null;
+	}
+
+	return (
+		<>
+			<PopoverMenuItem onClick={ generateQRCode } icon={ <Icon icon={ qrIcon } /> }>
```

Comment:
Small thing, but might help readability: Would it be possible to wrap the `<Icon>` part already in the `qrIcon` const?

Reply:
Agree. I'm on it!

Reply:
Done